### DISCUSSION TITLE
About model compression(such as model quantization)

### DISCUSSION THREAD

**Discussion by hadesfgh** [open-mmlab/mmediting#1195] [Models]
Have your team ever tried to compress the super-resolution models, such as realbasicvsr. Or does open-mmlab have any relevant tools to compress thees models.

**Comment by zengyh1900** [comment]
Hi @hadesfgh 

Yes! OpenMMLab has mmrazor! Please have a look here: https://github.com/open-mmlab/mmrazor

MMRazor is a model compression toolkit for model slimming and AutoML, which includes 3 mainstream technologies:

Neural Architecture Search (NAS)
Pruning
Knowledge Distillation (KD)
Quantization (in the next release)

Please let me know if you would like to contribute some compressed super-resolution models to MMEditing! :)
